cppguide.html

```diff
@@ -4360,12 +4360,15 @@ <h4>Common Variable names</h4>
 
 <h4>Class Data Members</h4>
 
-<p>Data members of classes, both static and non-static, are
+<p>Private and protected data members of classes, both static and non-static, are
```

Comment:
```suggestion
<p><del>Data</del><strong>Private and protected</strong> data members of classes, both static and non-static, are
```
Deviations from the official google style guide (which this guide is based on) are always marked by ~old~ **new**.